content/compose/networking.md

```diff
@@ -9,8 +9,7 @@ title: Networking in Compose
 By default Compose sets up a single
 [network](../engine/reference/commandline/network_create.md) for your app. Each
 container for a service joins the default network and is both reachable by
-other containers on that network, and discoverable by them at a hostname
-identical to the container name.
+other containers on that network, and discoverable by the container name.
```

Comment:
#18904 is suggesting that the hostname is the service name, not the container name. E.g. the container name in the following example would be `svcs-web-1`, but the service would be discoverable on the bridge network via the `web` hostname.

```yaml
services:
  web:
    image: nginx:alpine
    ports:
      - 80:80
  sh:
    depends_on:
      - web
    build:
      context: .
      dockerfile_inline: |
        FROM alpine
        RUN apk add curl
        CMD curl http://web:80
```